package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@ni/javascript-styleguide",
+  "version": "1.0.0",
+  "description": "NI's internal and external JavaScript and TypeScript style guide",
+  "private": true,
+  "scripts": {
+    "eslint-config:install": "cd packages/eslint-config && npm install",
+    "eslint-config:publish": "cd packages/eslint-config && npm publish",
+    "eslint-config:test": "cd packages/eslint-config; npm test",
+    "preinstall": "npm run eslint-config:install",
+    "publish:packages": "npm run eslint-config:publish",
```

Comment:
Have you guys talked about adding a pipeline to do the publish in a future submission? Or would it be a manual process?

Reply:
I think @rajsite  is interested in taking this on. I prefer someone more familiar with the GitHub setup than myself, or someone that will put the time in. I piped it to the root script API at the least. It's based on AirBnB and Google.

Reply:
@rajsite will do this